### ISSUE TITLE
Renaming files by adding spaces causes havoc 

### ISSUE THREAD

**Issue by apblack** [gracelang/grace-web-editor#49]
If I change the name of a file by editing the name in the title-bar of the editor pane, _and the change is to insert one or more spaces_, then the name change seems to take effect in localStorage, but not in the IDE elements that populate the file list. So the file becomes inaccessible until the IDE is refreshed.

If the name change is the insertion of other characters, all seems well!

**Comment by Dmitri-2** [comment]
Could you clarify what you mean by the name change taking effect "in the directory tree, but not in the IDE elements that populate the file list." 

Do you mean that the changes occur in localStorage, but not in the IDE sidebar with the files and directories?

**Comment by apblack** [comment]
> Do you mean that the changes occur in localStorage, but not in the IDE sidebar with the files and directories?

Yes, that's what I meant.  I edited the original issue.  Sorry, I was trying to type this while students were asking me questions.

**Comment by Dmitri-2** [comment]
Okay, thank you! I haven't been able to replicate the issue, which is why I asked for clarification. I have tried this in Chrome, Firefox and Safari, but haven't been able to re-create the problem.  
Here are the steps that I took to try to replicate it: 

1. Open a file (`FileToOpen.grace`).
2. Edit the name in the title bar of the editor, adding two (or more) 
spaces to change the name to `File  ToOpen.grace`
3. Switch to another file and then reopen `File  ToOpen.grace` – works for me
 Edit the renamed file (`File  ToOpen.grace`) in the editor – works for me 

Is there a step I am not taking to re-create this issue?
